### DISCUSSION TITLE
version>= will install newest or closest

### DISCUSSION THREAD

**Discussion by yonghenghuanmie** [microsoft/vcpkg#28541] [Q&A]
Use vcpkg to install dependency, which version will it install?
If newest, then what if new version doesn't compatible to old version and there is no way to choose specific version?

**Comment by autoantwort** [comment]
https://learn.microsoft.com/en-us/vcpkg/users/versioning.concepts

**Reply by yonghenghuanmie** [reply]
Got it, thank you.
